src/Raven.Server/Config/Categories/TrafficWatchConfiguration.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using System.ComponentModel;
+using Raven.Client.Documents.Changes;
+using Raven.Client.ServerWide.Operations.TrafficWatch;
+using Raven.Server.Config.Attributes;
+using Raven.Server.Config.Settings;
+using Sparrow;
+using Size = Sparrow.Size;
+
+namespace Raven.Server.Config.Categories;
+
+[ConfigurationCategory(ConfigurationCategoryType.TrafficWatch)]
+public class TrafficWatchConfiguration : ConfigurationCategory
+{
+    [Description("Traffic Watch logging mode.")]
+    [DefaultValue(TrafficWatchMode.Off)]
+    [ConfigurationEntry("TrafficWatch.Mode", ConfigurationEntryScope.ServerWideOnly)]
+    public TrafficWatchMode TrafficWatchMode { get; set; }
+
+    [Description("A semicolon-separated list of database names by which the Traffic Watch logging entities will be filtered. If not specified, Traffic Watch entities of all databases will be included. Example list: \"test-database;another-database;the-third-database\".")]
+    [DefaultValue(null)]
+    [ConfigurationEntry("TrafficWatch.Databases", ConfigurationEntryScope.ServerWideOnly)]
+    public List<string> Databases { get; set; }
+
+    [Description("A semicolon-separated list of response status codes by which the Traffic Watch logging entities will be filtered. If not specified, Traffic Watch entities with any response status code will be included. Example list: \"200;500;404\".")]
+    [DefaultValue(null)]
+    [ConfigurationEntry("TrafficWatch.StatusCodes", ConfigurationEntryScope.ServerWideOnly)]
+    public List<int> StatusCodes { get; set; }
+    
+    [Description("Minimum response size by which the Traffic Watch logging entities will be filtered.")]
+    [DefaultValue(0)]
+    [MinValue(0)]
+    [SizeUnit(SizeUnit.Bytes)]
+    [ConfigurationEntry("TrafficWatch.MinimumResponseSizeInBytes", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
the configuration entry key is MinimumResponseSizeInBytes, but the property name (since it is Size) should be called: MinimumResponseSize